packages/app-project/src/screens/ProjectAboutPage/ProjectAboutPageConnector.js

```diff
@@ -0,0 +1,62 @@
+import { observer, MobXProviderContext } from 'mobx-react'
+import React, { useContext } from 'react'
+import { arrayOf, bool, object, shape, string } from 'prop-types'
+
+import ProjectAboutPage from './ProjectAboutPage'
+
+/**
+  This is for enzyme tests.
+*/
+function useStoreContext (testStore) {
+  const { store } = testStore || useContext(MobXProviderContext)
```

Comment:
I have a non-blocking optional suggestion about using sinon's stub mocking for the tests, but it'd require changing the `useContext` here to `React.useContext`. More details are in a comment for the spec file.